app/server/tests/functions/communityFunctions.js

```diff
@@ -54,7 +54,7 @@ class CommunityFunctions {
    * @param {String}  token       JWT that authenticates the user.
    * @returns                     Response from API. Body contains object with community info.
    */
-  createCommunity(title, description, rules, tags, token) {
```

Comment:
why do we not need the rules and tags anymore?

Reply:
The endpoint on swagger doesn't include them. Not sure when this changed, but could only get it to create the community without them.